### PR TITLE
Profile email design

### PR DESCRIPTION
**Pull request by reactexcel**

### CODE REVIEW

File: src/components/CardTrail.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+import { View, StyleSheet } from "react-native";
+
+const CardTrail = () => {
+  return (
+    <Fragment>
+      <View
+        style={{
+          width: "70%",
+          flex: 0.020,
+          backgroundColor: "#39487a",
```
Comment (excellencetechnologies) [inline src/components/CardTrail.js]: wrong

File: src/components/CardTrail.js [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+          borderTopRightRadius: 15,
+          borderTopLeftRadius: 15
+        }}
+      />
+      <View
+        style={{
+          width: "75%",
+          flex: 0.020,
+          backgroundColor: "#5a6799",
```
Comment (excellencetechnologies) [inline src/components/CardTrail.js]: wrong

File: src/components/CardTrail.js [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+          borderTopRightRadius: 15,
+          borderTopLeftRadius: 15
+        }}
+      />
+      <View
+        style={{
+          width: "80%",
+          flex: 0.020,
+          backgroundColor: "#93a0cb",
```
Comment (excellencetechnologies) [inline src/components/CardTrail.js]: wrong

File: src/components/JobOpeningAboutUs.js [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+import { ABOUT_US } from "../helper/constant";
+
+const ProfileView = props => {
+  return (
+    <View style={styles.descriptionView}>
+      <Text style={styles.textStyle}>About Excellence Technologies</Text>
+      <View style={styles.aboutUsView}>
+        <Text style={styles.aboutUs}>{ABOUT_US}</Text>
+      </View>
+      {/* <View style={{ alignContent: "center" }}>
```
Comment (excellencetechnologies) [inline src/components/JobOpeningAboutUs.js]: what is this

File: src/components/JobOpeningAboutUs.js [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+const styles = StyleSheet.create({
+  descriptionView: {
+    flexBasis: "50%",
+    backgroundColor: "white",
+    alignItems: "center",
+    justifyContent: "center",
+    paddingHorizontal: 10
+  },
+  textStyle: {
+    fontFamily: "Montserrat-SemiBold",
```
Comment (excellencetechnologies) [inline src/components/JobOpeningAboutUs.js]: wrong

File: src/components/JobSalaryDetails.js [hunk 6]
```diff
@@ -0,0 +1,41 @@
...
+    <View style={styles.viewDesign}>
+      {props.children}
+      <View style={styles.iconView}>
+        <View style={styles.viewRow}>
+          <FontAwesome
+            name="briefcase"
+            color={COLOR.Red}
+            style={styles.viewIcon}
+          />
+          <Text style={styles.rowText}>0 - 1 years</Text>
```
Comment (excellencetechnologies) [inline src/components/JobSalaryDetails.js]: don't hard code it here. pass via props

File: src/components/JobSalaryDetails.js [hunk 7]
```diff
@@ -0,0 +1,41 @@
...
+            name="map-pin"
+            color={COLOR.Red}
+            style={styles.viewIcon}
+          />
+          <Text style={styles.rowText}>1,25,000 - 2,00,000 PA</Text>
+        </View>
+      </View>
+    </View>
+  );
+};
```
Comment (excellencetechnologies) [inline src/components/JobSalaryDetails.js]: there are no required props? wrong

File: src/components/JobOpeningAboutUs.js (hunk 4 above)
Comment (reactexcel) [reply src/components/JobOpeningAboutUs.js]: sir it is a section in Job Openings page

File: src/components/JobSalaryDetails.js (hunk 6 above)
Comment (reactexcel) [reply src/components/JobSalaryDetails.js]: sir, this component is just a single line, used in other component with the same data that's why it is hardcoded.

**Pull request merged**